### PR TITLE
feature/Header: styling added

### PR DESCRIPTION
**Pull request by VilmaIkonen**
src: Header.css file added and changes made to App.css (root, color variants) and Header.js (import)

### CODE REVIEW

File: frontend/package-lock.json [hunk 1]
```diff
@@ -1,18474 +1,8 @@
 {
```
Comment (vesasuikkola) [inline frontend/package-lock.json]: You  probably did not mean to change our package-lock?

**Review by vesasuikkola — CHANGES_REQUESTED** [review]
Please check file inline comment

File: frontend/package-lock.json (hunk 1 above)
Comment (VilmaIkonen) [reply frontend/package-lock.json]: I did not mean to change anything there.

Comment (vesasuikkola) [reply frontend/package-lock.json]: Resolved with 0d9cdab8155d4ba63ec44c40c11fb7dbdd808952

**Review by vesasuikkola — DISMISSED** [review]

**Review by SaharPerveen — CHANGES_REQUESTED** [review]
i am unable to see user icon.

**Review by SaharPerveen — APPROVED** [review]
user icon added in PBI.

**Review by vesasuikkola — APPROVED** [review]

**Pull request merged**